### ISSUE TITLE
I was using your gan model for other images that are high resolution (512,512,3) so when executing this line of code sequentially {train_dataset = tf.data.Dataset.from_tensor_slices(training_data).shuffle(BUFFER_SIZE).batch(BATCH_SIZE)}........ i got the error.....  Cannot create a tensor proto whose content is larger than 2GB.

### ISSUE THREAD

**Issue by ALI7861111** [jeffheaton/t81_558_deep_learning#52]

**Comment by jeffheaton** [comment]
This is really meant just as a starter.  You will likely have to modify the code to support really larger sizes.  You might want to use StyleGAN.

**Comment by ALI7861111** [comment]
Thanks a lot for your reply >  :)
